packages/api/AlvTimeWebApi/Validators/QuarterHour.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.ComponentModel.DataAnnotations;
+using AlvTimeWebApi.Requests;
+
+namespace AlvTimeWebApi.Validators
```

Comment:
Vi snakket forbi hverandre tror jeg :D Denne klassen kan fortsatt ligge i Business, i og med at det er en business-validering, men selve attributten kan ligge på requesten

Reply:
Her tror jeg du må forklare meg nærmere. Business prosjektet skal vel ikke vite hvordan Requesten ser ut? Hvis jeg har valideringen på Requesten i Business prosjektet må jeg jo referere til Request modellen i WebApi. Blir det riktig da?

Reply:
Request modellen ligger i WebApi, QuarterHour.cs ligger i Business. [QuarterHour] ligger på request.

Reply:
Nei, sorry. Jeg surrer. QuarterHour ser jo PayoutRequest. La meg tenke litt

Reply:
Vet du hva vi gjør? Dropp hele valideringsattributten. I PayoutService.cs så kjøres det en ValidatePayout-metode, legg den logikken inn der i stedet. Da får vi API request i API-laget, og business-logikk i business-laget.

Reply:
Da er valideringen for Payout lagt til i metoden `ValidatePayout`, og attributten ligger som tidligere på CreateTimeEntryDto. Var det slik du mente?